tests/test_populate.py

```diff
@@ -71,9 +70,22 @@ def test_scan_info_populate_scanbox_3D(testdata_paths, pipeline, scan_info):
 
 def test_processing_populate(processing, pipeline):
     imaging = pipeline['imaging']
+    
     assert len(imaging.Processing()) == 5
 
 
+def test_processing_populate_trigger_suite2p_2D(trigger_processing_suite2p_2D, pipeline):
+    imaging = pipeline['imaging']
+    assert len(imaging.Processing()) == 1
+
+    from element_interface.suite2p_loader import Suite2p
+    get_imaging_root_data_dir = pipeline['get_imaging_root_data_dir']
+    suite2p_dir = (get_imaging_root_data_dir() / 'demo/subject1_20200609DT170519_0_0/suite2p').as_posix()
+    Suite2p(suite2p_dir)
```

Comment:
Should we add an assert statement here?  And possibly assert that the downstream tables populated correctly?

Reply:
After discussion with @tdincer, running `Suite2p(suite2p_dir)` is sufficient to ensure the files are created and load correctly.